src/pick/index.ts

```diff
@@ -0,0 +1,9 @@
+export default function pick<
+  ObjectType extends {},
+  Key extends keyof ObjectType
+>(object: ObjectType, keys: Key[]): Pick<ObjectType, Key> {
```

Comment:
I'd suggest an alternative typing here, like so

```suggestion
>(object: ObjectType, keys: [Key, ...Key[]]): Pick<ObjectType, Key> {
```

This way TS users will be forced to provide `keys` array with at least one key.

Below is the error that user will get if using this typing and providing empty array for `keys`

![image](https://user-images.githubusercontent.com/769698/82357802-45d49200-9a0e-11ea-9e0a-64f6320f3403.png)

Reply:
Wow, great trick!